ipn/ipnlocal/local.go

```diff
@@ -592,6 +593,14 @@ func (b *LocalBackend) updateStatus(sb *ipnstate.StatusBuilder, extraLocked func
 				if c := sn.Capabilities; len(c) > 0 {
 					ss.Capabilities = append([]string(nil), c...)
 				}
+				if sn.Tags != nil {
```

Comment:
We should also add PrimaryRoutes

Reply:
or maybe we should consolidate this with `populatePeerStatusLocked` to not have a lot of duplicated code.

Reply:
Done